index.bs

```diff
@@ -982,42 +950,41 @@ WebGL Context Compatiblity {#contextcompatibility}
+In order for a WebGL context to be used as a source for XR imagery it must be created on a <dfn>compatible graphics adapter</dfn> for the [=XR device=]. What is considered a [=compatible graphics adapter=] is platform dependent, but is understood to mean that the graphics adapter can supply imagery to the [=XR device=] without undue latency. If a WebGL context was not already created on the [=compatible graphics adapter=], it typically must be re-created on the adapter in question before it can be used with an {{XRWebGLLayer}}. Once the [=XR compatible=] boolean is set, the context can be used with layers for any {{XRSession}} requested from that [=XR device=].
 
-Note: On an XR platform with a single GPU, it can safely be assumed that the GPU is compatible with the {{XRDevice}}s advertised by the platform, and thus any hardware accelerated WebGL contexts are compatible as well. On PCs with both an integrated and discreet GPU the discreet GPU is often considered the [=compatible graphics adapter=] since it generally a higher performance chip. On desktop PCs with multiple graphics adapters installed, the one with {{XRDevice}} physically connected to it is likely to be considered the [=compatible graphics adapter=].
+Note: On an XR platform with a single GPU, it can safely be assumed that the GPU is compatible with the [=XR device=]s advertised by the platform, and thus any hardware accelerated WebGL contexts are compatible as well. On PCs with both an integrated and discreet GPU the discreet GPU is often considered the [=compatible graphics adapter=] since it generally a higher performance chip. On desktop PCs with multiple graphics adapters installed, the one with the [=XR device=] physically connected to it is likely to be considered the [=compatible graphics adapter=].
 
-The [=compatible XR device=] can be set either at context creation time or after context creation, potentially incurring a context loss. To set the [=compatible XR device=] at context creation time, an {{XRDevice}} is supplied to the {{compatibleXRDevice}} context creation attibute dictionary when requesting a WebGL context.
+The [=XR compatible=] boolean can be set either at context creation time or after context creation, potentially incurring a context loss. To set the [=XR compatible=] boolean at context creation time, the {{xrCompatible}} context creation attibute must be set to <code>true</code> when requesting a WebGL context.
```

Comment:
Not specifically related to your change, but I can't recall.... did we evaluate the fingerprinting risks associated with setting a context compatibility?

Reply:
I don't believe we did, but I also can't think of much in the way of risk? Is there a specific fingerprinting avenue that you had in mind? (The fact that you MUST set the context compatibility bit whether or not the context would have been created on the right GPU anyway seems like it mitigates most issues that could have come up with it.)

Reply:
The scenario I was thinking of is: 
1. Creates a canvas and get the context
1. Peep at the GPU info
1. Sets the context for XR compatibility
1. Peep at the GPU info again

On desktop machines with the XR device plugged into a secondary GPU, this would enable a developer to know that there is more than one GPU without actually engaging the XR hardware.

Are we worried about this?

Reply:
Done. #429